LeanplumTests/LPSetVarsApiTest.m

```diff
@@ -0,0 +1,164 @@
+//
+//  LPSetVarsApiTest.m
+//  LeanplumTests
+//
+//  Created by Mayank Sanganeria on 5/22/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <OHHTTPStubs/OHHTTPStubs.h>
+#import <OHHTTPStubs/OHPathHelpers.h>
+#import "LPSetVarsApi.h"
+#import "LPAPIConfig.h"
+#import "LPConstants.h"
+#import "LPTestHelper.h"
+
+@interface LPSetVarsApiTest : XCTestCase
+
+@end
+
+@implementation LPSetVarsApiTest
+
+- (void)setUp {
+    [super setUp];
+    [LPTestHelper setup];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    [OHHTTPStubs removeAllStubs];
+}
+
+- (void)testSetVarsApi {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    [LPSetVarsApi setVarsWithParameters:nil success:^ {
+        [expectation fulfill];
+    } failure:^(NSError *error) {
+    }];
+    
+    [self waitForExpectationsWithTimeout:30.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+- (void)testSetVarsApiWithParameters {
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Query timed out."];
+    NSDictionary *params = @{ @"testKey": @"testValue" };
+    [LPSetVarsApi setVarsWithParameters:params success:^ {
+        [expectation fulfill];
+    } failure:^(NSError *error) {
+    }];
+    
+    [self waitForExpectationsWithTimeout:10.0 handler:^(NSError *error) {
+        if (error) {
+            NSLog(@"Error: %@", error);
+        }
+    }];
+}
+
+// TODO: verify why this works despite no dev key
```

Comment:
Can you please elaborate. This might be due to the similar issue , where some other test is setting keys.